lightning_examples/finetuning-scheduler/finetuning-scheduler.py

```diff
@@ -172,13 +172,22 @@
 # Import the `FinetuningScheduler` PyTorch Lightning extension module we want to use. This will import all necessary callbacks.
 import finetuning_scheduler as fts  # isort: split
 
-import lightning as L
+from lightning_utilities.core.imports import module_available  # noqa: I001
+
+if module_available("lightning"):
+    import lightning as L
+    from lightning.pytorch.callbacks import EarlyStopping, ModelCheckpoint
+    from lightning.pytorch.loggers.tensorboard import TensorBoardLogger
+    from lightning.pytorch.utilities import rank_zero_warn
+elif module_available("pytorch_lightning"):
+    import pytorch_lightning as L
+    from pytorch_lightning.callbacks import EarlyStopping, ModelCheckpoint
+    from pytorch_lightning.loggers.tensorboard import TensorBoardLogger
+    from pytorch_lightning.utilities import rank_zero_warn
```

Comment:
```suggestion
```
it won't work here until it is the same also `finetuning_scheduler`
you can test it by removing `lightning` from the requirements

Reply:
reverted to previous Lightning import approach as discussed and all is passing. 

FTS will install Lightning by default so we don't need to include the `lightning` dependency in the notebook-specific dependencies. 

Note if [one installs finetuning-scheduler](https://github.com/speediedan/finetuning-scheduler/blob/04350f123ef9f72a9ccdd90e9120803f01469742/README.md?plain=1#L81) with `export PACKAGE_NAME=pytorch` though it will install using the standalone `pytorch-lightning` dependency and the example should work with `pytorch-lightning` as well (when changing the imports manually or introspecting them as was done in a previous commit). 

Probably makes sense to keep the example as is for now since anyone installing FTS with that non-default mode will know the example imports will need to change and we'd prefer to keep the example as simple as possible. Thanks again!